src/main/java/org/folio/service/upload/UploadDefinitionServiceImpl.java

```diff
@@ -378,6 +380,22 @@ private Errors validateFreeSpace(UploadDefinition definition, Errors errors) {
     }, tenantId);
   }
 
+  @Override
+  public Future<Boolean> updateUploadDefinitionStatusIfAllFilesUploadingFailed(UploadDefinition uploadDefinition, UploadDefinition.Status status, String tenantId) {
```

Comment:
do we really need UploadDefinition.Status status argument for this method?

Reply:
at first the argument did not exist, but the second comment from @OleksiiKuzminov says that it's bad practice

Reply:
Can you just make method update status and call it if condition is true?